lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -0,0 +1,60 @@
+{
+  "workflows": [
+    {
+      "name": "medidate_depsoit",
+      "label": "Self-deposit workflow",
+      "description": "A multi-step workflow for self-depoist",
+      "actions": [
+         {
+           "name": "finalize_digitization",
```

Comment:
These don't seem like actions one would want in a mediated deposit workflow.  Am I off base here?